src/FixerDefinition/VersionSpecification.php

```diff
@@ -36,19 +36,21 @@
     public function __construct($minimum = null, $maximum = null)
     {
         if (null === $minimum && null === $maximum) {
-            throw new \InvalidArgumentException('Either minimum or maximum need to be specified');
+            throw new \InvalidArgumentException('Minimum and/or maximum need to be specified.');
```

Comment:
```
noun: OR; noun: or; plural noun: ors
1.
a logical operation which gives the value one if at least one operand has the value one, and otherwise gives a value of zero.
```

by google